pyproject.toml

```diff
@@ -52,6 +52,7 @@ urllib3 = "^1.26.16"
 sqlalchemy-utils = "^0.41.1"
 wintertoo = "^1.2.0"
 scipy = "^1.12.0"
+python-dotenv = "^1.0.1"
```

Comment:
I am trying to make mirar less picky re packages. Does this actually need the minor version 1.0.1, or is it likely to be fine with any python-dotenv < 2 ?

Reply:
I had a look through their release notes and added a logger.info line that requires >=0.21.0, so not anymore.

Basically it now needs >=0.21.0 and it should be fine with <2, which I think the "^" covers.

Reply:
Ah, sorry, you are right. I misunderstood the ^ use